assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -13,12 +13,14 @@
 .e-con-inner {
 
 	& > .elementor-widget-spacer {
+		display: flex;
```

Comment:
Do we really need to add display: flex?
This could cause Backwards Compatibility issues.
Have you tested it with Spacer widgets inside Sections as well?